### ISSUE TITLE
Grammar issue with timeline

### ISSUE THREAD

**Issue by SteadyStatus21** [immich-app/immich#9861]
### The bug

When looking at the homepage, for some reason the 1 year ago shows `1 years ago` when in reality it should show `1 year ago`. Saw this and it started to bug me, so I wanted to report it :)

For reference, the mobile app's text is correct.

![image](https://github.com/immich-app/immich/assets/73841304/59b87a33-68c0-4bb5-a039-2154a96d8bb2)

### The OS that Immich Server is running on

N/A

### Version of Immich Server

v1.105.1

### Version of Immich Mobile App

N/A

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
1. Load up Immich\'s web portal. 
2. View the timeline at the top of the page.
3. Notice how it says "1 years ago" instead of "1 year ago"
4. Point this out to the developers and contributors of Immich. (I really appreciate all of you btw :) )
```

**Comment by Snowknight26** [comment]
This was fixed with #9663. Can't reproduce on the latest commit from `main`.

**Comment by SteadyStatus21** [comment]
> This was fixed with #9663. Can't reproduce on the latest commit from `main`.

Gotcha. Thank you!
